doc/index.rst

```diff
@@ -1,9 +1,32 @@
-The Plutus Platform
+Plutus Tools Developer Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
+Plutus Tools are a collection of off-chain infrastructure resources. Plutus Tools are built 
+for developers who are authoring web-based applications that implement smart contracts on the 
+Cardano blockchain. 
+
+The Plutus Tools Team supports the underlying resources that developers need who are 
+writing full applications using Plutus in Haskell, including off-chain code. 
+The term “off-chain code” refers to the part of a contract application’s code which 
+runs off the chain, usually as a contract application. 
+
+Because there are so many potential use cases for what you may choose to create, you are not 
+confined to only one prescribed way of using Plutus Tools. Instead, you can access 
+the entire tool box where you can find a collection of utilities and components. Pick and choose 
+from the tools and examples that best suit your use case. 
+
+The Plutus Platform
+-------------------------------
+
+The Plutus Tools Repo (https://github.com/input-output-hk/plutus-apps) contains 
+the Plutus Platform, which consists of these components: 
+
+* Libraries which implement the Plutus Application Framework
+* A selection of end-to-end use cases written with the Plutus Application Framework
+* The Plutus Playground, a web-based playground for learning and writing basic Plutus Applications
```

Comment:
Didn't we decide we don't want to support this?

Reply:
I got these points from https://github.com/input-output-hk/plutus-apps#readme. Maybe this info is out of date. Are you referring to the Plutus Playground, or all 3 points here?

Reply:
@asutherlandus are we still supporting the playground?

Reply:
I'm removing this reference to the Plutus Playground.